docs/openapi.json

```diff
@@ -1242,4 +1256,4 @@
   },
   "swagger": "2.0",
   "tags": []
-}
+}
```

Comment:
Missing end-line. Perhaps we can amend `generate_openapi_spec.py` to avoid this? It happens to me all the time as well.